src/pages/City/Cities/SortedCities.tsx

```diff
@@ -203,7 +200,7 @@ const SortedCities = ( {switcher}: Props) => {
                 pageSize={pageSize}
                 total={total}
                 responsive
-                showLessItems
+                showSizeChanger={total < 20 ? false : true}
```

Comment:
You can simplify this ternary operator by writing "showSizeChanger={total >= 20}"

Reply:
Done